drivers/espi/espi_mchp_xec.c

```diff
@@ -861,22 +860,20 @@ static void espi_vwire_chanel_isr(struct device *dev)
 	struct espi_xec_data *data = (struct espi_xec_data *)(dev->driver_data);
 	const struct espi_xec_config *config = dev->config_info;
 	struct espi_event evt = { .evt_type = ESPI_BUS_EVENT_CHANNEL_READY,
-				  .evt_details = 0,
+				  .evt_details = ESPI_CHANNEL_VWIRE,
 				  .evt_data = 0 };
 	uint32_t status;
 
 	status = ESPI_IO_VW_REGS->VW_EN_STS;
 
 	if (status & MCHP_ESPI_VW_EN_STS_RO) {
 		ESPI_IO_VW_REGS->VW_RDY = 1;
-
+		evt.evt_data = 1;
 		/* VW channel interrupt can disabled at this point */
 		MCHP_GIRQ_ENCLR(config->bus_girq_id) = MCHP_ESPI_VW_EN_GIRQ_VAL;
 		send_slave_bootdone(dev);
```

Comment:
This assumes EC firmware has enabled SAF block.  Can Slave Boot Done VWires be sent upstream before the other channels are enabled by the master?

Reply:
Yes, slave boot done can be send before OOB and Flash channel handshake is completed.
Regarding SAF block is enabled. Assumption is that is initialized prior to release RSM_RST, prior to eSPI reset de-asserted.

1) SAF enablement
2) RSM_RST released by eSPI slave
3) eSPI reset de-asserted by eSPI master
-- eSPI handshake takes place
4) eSPI VW channel enabled by master
5) eSPI VW channel ready indicated by slave
6) Slave boot done VW  by slave  
7) eSPI flash channel enabled by master
8) eSPI flash channel ready indicated by slave

Once that is said, to give major flexibility, I can extend the CONFIG_ESPI_AUTOMATIC_WARNING_ACKNOWLEDGE mechanism in a separate PR so app can decided if slave boot done is send by the driver in (6) or app can send it after (8).

Reply:
Sounds good.